### ISSUE TITLE
Plugin 'org.apache.maven.plugins:maven-antrun-plugin:3.0.0' not found

### ISSUE THREAD

**Issue by Assambra** [youngmonkeys/ezyfox-server#107]
**Describe the bug**
I use the server from the EzySmashers demo project and get two errors in File pom.xml:
Plugin 'org.apache.maven.plugins:maven-antrun-plugin:3.0.0' not found:84
Plugin 'org.apache.maven.plugins:maven-antrun-plugin:3.0.0' not found:85

**To Reproduce**
1. Used IDE : IntelliJ IDEA Community Edition 2023.1.2
2. Java version: 11.0.16.1
3. maven-3.9.2
4. See the error

**Expected behavior**
No errors

**Screenshots**
no screenshots

**Desktop (please complete the following information):**
mvn -v
Apache Maven 3.9.2 (c9616018c7a021c1c39be70fb2843d6f5f9b8a1c)
Maven home: C:\apache-maven-3.9.2
Java version: 11.0.16.1, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-11.0.16.1
Default locale: de_DE, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"

**Smartphone (please complete the following information):**
no information

**Additional context**
Is there some incompatibility or do i need other Java, Maven versions?

**Comment by Assambra** [comment]
I fixed by myself if sombody has the same problems:

1. I installed the IntelliJ IDEA Community Edition 2023.1.2 this or some older Java installation installed the Java version 11 on my system.
2. Install Java 8
3. Check in Windows with "java -version" your Java version if the output something like java version "1.8.x" thats good.
4. If not check the Path variable in windows. In my case there was two entries one to the Java 11 version and the other one to the Java 8 version with a different location. I deleted the entry for the Java 11 version and let in my case the entry C:\Program Files (x86)\Common Files\Oracle\Java\javapath.
5. I dont now if it is nessasary but i also created the JAVA_HOME variable with the value in my case C:\Program Files\Java\jdk-1.8 
6. After that i rebooted my system to take the effect.
7. IntelliJ IDEA : File -> Project Structure -> Project Settings -> Project -> SDK -> Down Arrow -> Add SDK -> add the location to your Java 8 files. in my case: C:\Program Files\Java\jdk-1.8
